ocs_ci/ocs/resources/drpc.py

```
@@ -143,6 +143,64 @@ def get_last_kubeobject_protection_time(self):
+
+        Returns:
+            str: JSON merge-patch string ready to pass to drpc_obj.patch()
+        """
+        annotations = self.get().get("metadata", {}).get("annotations", {}) or {}
+        last_action = annotations.get(constants.DRPC_LAST_ACTION_ANNOTATION, "")
+        last_cluster = annotations.get(
+            constants.DRPC_LAST_APP_DEPLOYMENT_CLUSTER_ANNOTATION, ""
+        )
+        logger.info(
+            f"DRPC '{self.resource_name}': last-action={last_action!r}, "
+            f"last-app-deployment-cluster={last_cluster!r}"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)